### PR TITLE
Added HCA Tissue Atlas, HCA Tissue Atlas Version, Project Integrated Status, and HCA Project uuid field. Fixes #1517

### LINKED ISSUE #1517: Update project tab with four metadata entities for grouping Tissue Atlas projects
### For which schema is a change/update being suggested?
Create a new 'hca_bionetwork' module and add it to project.json

### What should the change/update be?
Creating four fields as part of the 'hca_bionetwork' module and adding them to project.json

#### 1. HCA Bionetworks
I would like to add a new field - `Project.hca_bionetwork.name` - to this schema. This will allow data contributors to denote if the project is part of a HCA bionetwork and if so, which one.

This update constitutes a minor change to the schema(s) it affects.

```
Field name: project.hca_bionetwork.name
Field description: A field describing if the project is part of an HCA Bionetwork (e.g. Kidney). Enter ‘No’ if the project is not part of an HCA Bionetwork
Field type: array
Required: no
Examples: Kidney; Lung; No
CV or enum: enum
```

#### 2. HCA Tissue Atlas
I would like to add a new field - `Project.hca_bionetwork.hca_tissue_atlas` - to this schema. This will allow data contributors to denote if the project is part of a HCA Tissue Atlas and if so, which one. In this case, a Tissue Atlas represents a singular h5ad file. 

This update constitutes a minor change to the schema(s) it affects.

```
Field name: project.hca_bionetwork.hca_tissue_atlas
Field description: A field describing if the project is part of a HCA Tissue Atlas (e.g. Kidney). 
Field type: string, array
Required: no
Examples: Brain Alzheimer's Atlas 
CV or enum: Free Text
```

#### 3. HCA Tissue Atlas Version
I would like to add a new field - `Project.hca_bionetwork.hca_tissue_atlas_version` - to this schema. This will allow data contributors to denote what version of the Tissue Atlas is associated with this publication (e.g. v1.0; v2.0)

This update constitutes a minor change to the schema(s) it affects.

```
Field name: project.hca_bionetwork.hca_tissue_atlas_version
Field description: A field describing the version of the HCA Tissue Atlas is associated with the publication (e.g. v1.0; v2.0)
Field type: string, array
Required: no
Examples: 'v1.0; v2.0'
CV or enum: REGEX
```

#### 2. Integrated project
I would like to add a new field - `Project.hca_bionetwork.atlas_project` - to this schema. This will allow data contributors to denote if the publication is the HCA Tissue Atlas publication which integrates data from other projects within the atlas.

This update constitutes a minor change to the schema(s) it affects.

```
Field name: project.hca_bionetwork.atlas_project
Field description: A field describing if the project integrates data from other publications. 
Enter ‘Yes’ if this project is a HCA Tissue Atlas and it integrates data from all other datasets. Enter ‘No’, if this project's data is being integrated.
Field type: string
Required: no
Examples: 'Yes; No'
CV or enum: enum
```

### Motivation
Biological Network Data Contributors need to be able to denote if 
a) Their project is part of a HCA Bionetwork
b) Their project is part of a HCA Tissue Atlas
c) What version of the HCA Tissue Atlas the project is a part of
b) Whether the project is the 'Atlas Project' which draws data from other projects, or the project is a project from which data is being integrated

### PR DESCRIPTION
**Pull request by Wkt8**
### Release notes

For module/project/publication.json schema:
- Added `hca_tissue_atlas` field
- Added `integrated_status` field
- Added `hca_tissue_atlas_version` field
- Added `hca_project_uuid` field
 
### Why are these changes needed?

This change is needed for data contributors and consumers, to identify projects part of an official HCA Tissue Atlas, to identify the Atlas Project which integrates other analysis files, denote the version of the HCA Tissue Atlas, and describe the uuid of the associated HCA Project 

### Reviews requested

- Need Reviewers to approve because this is a minor update

We have planned the four metadata schema fields to explicitly describe the metadata of the project. [Wrangler meeting notes](https://docs.google.com/document/d/1N3dJA5dQuJcY5pxhT2pNgwZptKfdH3BSsqZSnWD0nUo/edit#)
[Example spreadsheet](https://docs.google.com/spreadsheets/d/1CiyCJZ96lu3PBLr7RAgbz08h1qA66n0Z/edit?usp=sharing&ouid=114069941208994528669&rtpof=true&sd=true)

### CODE REVIEW

File: json_schema/module/project/publication.json [hunk 1]
```diff
@@ -62,6 +62,50 @@
             "user_friendly": "Official HCA Publication",
             "guidelines": "Should be one of: yes, or no.",
             "example": "yes; no"
+        },
+        "hca_tissue_atlas": {
+            "description": "A field describing if the project is part of an HCA Tissue Atlas (e.g. Kidney). Enter ‘No’ if the project is not part of an HCA Tissue Atlas.",
```
Comment (idazucchi) [inline json_schema/module/project/publication.json:67]:
```suggestion
            "description": "A field describing if the project is part of an official HCA Tissue Atlas (e.g. Kidney). Enter ‘No’ if the project is not part of an official HCA Tissue Atlas.",
```

File: json_schema/module/project/publication.json [hunk 2]
```diff
@@ -62,6 +62,50 @@
...
+                "Heart",
+                "Immune",
+                "Kidney",
+                "Liver",
+                "Lung",
+                "Musculoskeletal",
+                "Nervous System",
+                "Oral & Craniofacial",
+                "Organoid",
+                "Pancrease",
```
Comment (idazucchi) [inline json_schema/module/project/publication.json:86]:
```suggestion
                "Pancreas",
```

File: json_schema/module/project/publication.json [hunk 3]
```diff
@@ -62,6 +62,50 @@
...
+                "Reproduction",
+                "Skin",
+                "No"
+            ],
+            "user_friendly": "Official HCA Tissue Atlas",
+            "guidelines": "Should be one of the networks from https://www.humancellatlas.org/biological-networks/ or 'No' if not a network",
+            "example": "Kidney; Lung; No"
+        },
+        "integrated_status": {
+            "description": "A field describing if the HCA Tissue Atlas project integrates data from other projects. ",
```
Comment (idazucchi) [inline json_schema/module/project/publication.json:96]:
```suggestion
            "description": "A field describing if the HCA Tissue Atlas project is the result of dataset integration. ",
```

**Review by idazucchi — COMMENTED** [review]
Hi! 
The PR looks good but I have a question about the versioning. Does it have to be in the format vX.X? Wouldn't a single number be enough?

If we want to keep the dot notation I would suggest adding a reular expression to avoid spelling variants (spaces, capitalisation and other mistakes)

**Review by anu-shiva — COMMENTED** [review]
Do we also want to add the year-month of the release (as it makes it a bit simpler  for the contributors to understand the  releases in addition to the atlas version )

File: json_schema/module/project/publication.json [hunk 4]
```diff
@@ -62,6 +62,59 @@
...
+            "type": "string",
+            "enum": ["Atlas Project","No"],
+            "user_friendly": "Project Integrated Status",
+            "guidelines": "Enter ‘Atlas Project’ if this project is one of the HCA Tissue Atlases and it integrates data from all other datasets. Enter ‘No’, if this project's data is being integrated.",
+            "example": "Atlas Project; No"
+        },
+        "hca_tissue_atlas_version": {
+            "description": "A field describing which version of the HCA Tissue Atlas is associated with the publication (e.g. v1.0; v2.0)",
+            "type": "string",
+            "pattern": "v[0-9].[0-9]",
```
Comment (ESapenaVentura) [inline json_schema/module/project/publication.json:106]:
```suggestion
            "pattern": "^v[0-9]+\\.[0-9]+$",
```

**Review by ESapenaVentura — CHANGES_REQUESTED** [review]

**Comment by Wkt8** [comment]
Closing PR without merging and creating a new one due to conversations between EBI wranglers, UCSC wranglers, and Dave.

**Pull request closed without merging**